lib/foreman_remote_execution_core/script_runner.rb

```diff
@@ -279,5 +282,17 @@ def check_expecting_disconnect
         @expecting_disconnect = true
       end
     end
+
+    def available_authentication_methods
+      methods = %w(pubkey) # Always use pubkey auth as fallback
+      if settings.kerberos_auth
```

Comment:
I'm getting `NoMethodError - undefined method `kerberos_auth' for #<Hash:0x005609203c17f0>`, tried with smart proxy 1.15 as well as develop, with latest versions of plugins. Am I missing something?